02week/blog/css/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  margin: 10px 10px;
+}
+
+/* images */
+img {
+  border-radius: 50%;
+  border-style: solid;
+  height: 50%;
+  width: 25%;
+}
+/* headings */
+body, html {
+  height: 100%;
+}
+header {
+  border-radius: 20px;
+  border: black solid;
+  background-color: dodgerblue;
+  text-shadow: 2px 2px 5px black;
+  font-size: 70px;
+  color: white;
+  padding: 50px;
+  text-align: center;
+}
+h1 {
+  text-align: center;
+  font-size: 35px;
+  color: black;
+  border-radius: 20px;
+  background-color: lightblue;
+  border: solid black;
+  float: right;
+  width: 200px;
+  margin-top: 100px;
+  margin-right: 250px;
+  padding: 50px;
+}
+span.special:hover {
+    color: black;
+    font-size: 50px;
+    white-space: nowrap;
+    margin: 0;
+}
+span.other:hover {
```

Comment:
How would this be different if i wanted the hover on the entire blog article to trigger the change?